packages/ketcher-polymer-editor-react/src/components/shared/ui/Tabs/Tabs.tsx

```diff
@@ -29,44 +28,79 @@ function TabPanel({ children, value, index, ...other }) {
   )
 }
 
-function CustomTabs(props) {
+function a11yProps(index: number) {
+  return {
+    id: `simple-tab-${index}`,
+    'aria-controls': `simple-tabpanel-${index}`
+  }
+}
+
+function CustomTabs(props): ReactElement {
   const [tabIndex, setTabIndex] = useState(0)
-  const { tabs, contentClassName, className } = props
+  const { tabs, contentStyle, tabsStyle } = props
   const tabPanel = tabs[tabIndex]
   const Component = tabPanel?.component
   const componentProps = tabPanel?.props
 
-  const handleChange = (event, newTabIndex) => {
+  const handleChange = (_event, newTabIndex) => {
     setTabIndex(newTabIndex)
   }
 
+  const StyledTabs = styled(Tabs)<TabsProps>({
```

Comment:
We don't need to specify `<TabsProps>` explicitly, `styled()` infers props types from whatever component is passed to it.

We can use this type annotation if we want to change these props types, so it's optional.